examples/default-tags-yaml/Pulumi.yaml

```diff
@@ -10,4 +10,4 @@ resources:
   my-bucket:
     type: aws:s3:Bucket
 outputs:
-  bucketTags: ${my-bucket.tagsAll}
+  bucketTags: ${my-bucket.tags}
```

Comment:
What's the reasoning to switch `tagsAll` to `tags` in the test? Does `tagsAll` still work too?

Reply:
Because we are the ones setting `tags`. `tagsAll` is managed by TF (derived from `tags`). They are equivalent. We can tests for both.